lib/tensorflow/graph.rb

```diff
@@ -86,18 +87,45 @@ def define_op(opName, name , input, device, attrs)
     node
   end
 
+  #
+  # Creates a constant Tensor that is added to the graph with a specified name.
+  # Official documentation of {tf.constant}[https://www.tensorflow.org/versions/r0.9/api_docs/python/constant_op.html#constant].
+  #
+  def constant(name, data, type)
+    tensor = Tensorflow::Tensor.new(data, type)
+    self.constants = {name => tensor}
+    self.define_op("Const", name, nil, "", {"dtype" => tensor.type_num, "value" => tensor})
```

Comment:
About this call:
1. When the last argument is hash, you can omit its parens: `self.define_op("Const", name, nil, "", "dtype" => tensor.type_num, "value" => tensor)`
2. I'd prefer to have Symbol as a key for parameters wherever it is possible: `self.define_op("Const", name, nil, "", dtype: tensor.type_num, value: tensor)`, and even `define_op(:const, name, nil, "", dtype: tensor.type_num, value: tensor)`
3. `self.` is unnecesary here.